elements/lisk-codec/src/codec.ts

```diff
@@ -163,13 +163,21 @@ export class Codec {
 			if (properties === undefined) {
 				throw new Error('Invalid schema. Missing "properties" property');
 			}
+			for (const property of Object.values(properties)) {
+				if (!('fieldNumber' in property)) {
```

Comment:
I think we want to check `fieldNumber` exists and the value in the range 1 to 18’999. (https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md#field-name-and-field-number)